### ISSUE TITLE
Blank pages in preferences

### ISSUE THREAD

**Issue by TrueHatori** [e107inc/e107#1479]
I get for three days in the preferences for Comments / posting, File Uploading, Security & Protection, Javascript Framework and Advanced Features a blank page, so I can adjust anything. Why is that? What can I do?
Its on the radio site http://www.dragonland-radio.de
Everything else works correctly.

**Comment by tgtje** [comment]
More a question for e107help; just guessing here as it is looking a lot like before on v1 : theme error, or some php error (closing tag), or most likely a language error (make sure your german langpack in use is up to date as much as possible) I do see english (news dates > those should normally be german too) and german. (Else is look from outside, seems ok, so guessing..)

**Comment by TrueHatori** [comment]
The laguage verify say 0 errors, but in tab Plugins not all plugins displayed (e.g. links page), I will check this next week. Thanks.

**Comment by Moc** [comment]
Please make sure you are using the latest files from Github (also retrieve the latest version of the language pack if you are using a different language than English). This should resolve the issue.

**Comment by tgtje** [comment]
Quick help (works most cases when language troubles) > go into maintenance; switch system to use English, run tests.. if it works (what previous did not) > language issue ( please do NOT change content (txt inside pages etc..; else you might stumble into unwanted sideeffects... german has english terms etc..)

**Comment by TrueHatori** [comment]
Yes, this it is, thanks. In English is all correct, in German is the problem. Must I search for the terms :(

**Comment by Moc** [comment]
@TrueHatori Again, please make sure you are using:
- The latest files from Github 
- The latest version of the language pack (https://github.com/e107inc/e107-v2.x-Language-Packs)

I just tested this and here is no blank page in any of the preferences section. 

If the issue persists, please send me an email to e107@tijnkuyper.nl with admin login information so I can find the cause of the issue for you.

**Comment by TrueHatori** [comment]
No, never I use Bing-Files ;) I have translated most files, I do not overwrite with Bing. I should stop myself until I find the right and everything is running again. Until then, I may switch to English. I'll find it, do not hurry. I think it is in the files of the plugin link page. Since I installed, there is a problem. This is the only plugin that does not appear in Preferences -> Language packs -> Plugins.
